components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthUtil.java

```diff
@@ -856,6 +856,10 @@ private static boolean processTokenRevocation(Set<String> clientIds, Authenticat
             try {
                 Set<AccessTokenDO> accessTokenDOs;
                 try {
+                    if (LOG.isDebugEnabled()) {
+                        LOG.debug("Retrieving all ACTIVE or EXPIRED access tokens for the client: " + clientId
```

Comment:
Lets add userStoreDomain as well

Reply:
Addressed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2424/commits/0682abb603d75876afc501e9772748d8b916db2d